drivers/pinmux/Kconfig.cc2650

```diff
@@ -0,0 +1,16 @@
+# Apache-2.0 License
+#
+# Kconfig for TI CC2650 SoC pinmux driver.
+
+config PINMUX_CC2650
+	bool "Pinmux driver for CC2650 SoC"
+	depends on PINMUX && SOC_SERIES_CC2650
+	select GPIO
+	select GPIO_CC2650
```

Comment:
since pin mux depends on GPIO driver, the GPIO driver should be first in the sequence of commits.